docs/development/setup-vagrant.md

```diff
@@ -254,7 +276,7 @@ christie@win10 ~
 $ cd zulip
 
 christie@win10 ~/zulip
-$ vagrant up
+$ vagrant up --provider=docker
```

Comment:
This change would only be correct for Linux, so it can't go here.

Reply:
Fixed.